apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/DynamicLargeObjectApiLiveTest.java

```diff
@@ -0,0 +1,175 @@
+      awaitConsistency();
+      assertMegabyteAndETagMatches(regionId, containerName, name + "/2", etag2s);
+
+      List<Segment> segments = ImmutableList.<Segment> builder()
+            .add(Segment.builder().path(format("%s/%s/1", containerName, name)).etag(etag1s).sizeBytes(1024 * 1024)
+                  .build())
+            .add(Segment.builder().path(format("%s/%s/2", containerName, name)).etag(etag2s).sizeBytes(1024 * 1024)
+                  .build())
+            .build();
+
+      awaitConsistency();
+      String etagOfEtags = getApi().getDynamicLargeObjectApi(regionId, containerName).replaceManifest(name, segments,
+            ImmutableMap.of("myfoo", "Bar"), ImmutableMap.of("header1", "value1"));
+
+      assertNotNull(etagOfEtags);
+
+      awaitConsistency();
+
+      SwiftObject bigObject = getApi().getObjectApi(regionId, containerName).get(name);
+      assertThat(bigObject.getETag().equals(etagOfEtags));
+      assertThat(bigObject.getPayload().getContentMetadata().getContentLength().equals(Long.valueOf(2 * 1024 * 1024)));
+      assertThat(bigObject.getMetadata().equals(ImmutableMap.of("myfoo", "Bar")));
+
+      // segments are visible
+      assertThat(getApi().getContainerApi(regionId).get(containerName).getObjectCount().equals(Long.valueOf(3)));
+   }
+
+   protected void assertMegabyteAndETagMatches(String regionId, String containerName, String name, String etag1s) {
+      SwiftObject object1s = getApi().getObjectApi(regionId, containerName).get(name);
+      assertThat(object1s.getETag().equals(etag1s));
+      assertThat(object1s.getPayload().getContentMetadata().getContentLength().equals(Long.valueOf(1024 * 1024)));
+   }
+
+   protected void deleteAllObjectsInContainerDLO(String regionId, final String containerName) {
+       ObjectList objects = getApi().getObjectApi(regionId, containerName).list(new ListContainerOptions());
+      if (objects == null) {
+         return;
+      }
+      List<String> pathsToDelete = Lists.transform(objects, new Function<SwiftObject, String>() {
+         public String apply(SwiftObject input) {
+            return containerName + "/" + input.getName();
+         }
+      });
+      if (!pathsToDelete.isEmpty()) {
+         for (String name : pathsToDelete)
+            getApi().getObjectApi(regionId, containerName).delete(name);
+      }
+   }
+
+   @Override
+   @BeforeClass(groups = "live")
+   public void setup() {
+      super.setup();
+      for (String regionId : regions) {
+         boolean created = getApi().getContainerApi(regionId).create(defaultContainerName);
+         if (!created) {
+            deleteAllObjectsInContainer(regionId, defaultContainerName);
+         }
+      }
+
+     /* megOf1s = new byte[1024 * 1024];
+      megOf2s = new byte[1024 * 1024];
+
+      Arrays.fill(megOf1s, (byte) 1);
+      Arrays.fill(megOf2s, (byte) 2);*/
+      
+      megOf1s = TestUtils.randomByteSource().slice(0, 1048576);
```

Comment:
Move initialization of these constants to their declarations and mark `static final`.